Multisig/UI/Transaction/PaymentMethod/ChoosePaymentViewController.swift

```diff
@@ -13,6 +13,7 @@ class ChoosePaymentViewController: UIViewController, UITableViewDelegate, UITabl
     @IBOutlet private weak var tableView: UITableView!
     var relaysRemaining: Int = 0
     var relaysLimit: Int = 0
+    var userSelectedSigner = false
```

Comment:
As we discussed before, can we replace this bool with state variable

Reply:
this is not part of this pr. This flag is used and set outside of the dialog.